piptools/repositories/pypi.py

```diff
@@ -59,7 +59,6 @@ def __init__(self, pip_options, session):
             index_urls=index_urls,
             trusted_hosts=pip_options.trusted_hosts,
             allow_all_prereleases=pip_options.pre,
-            process_dependency_links=pip_options.process_dependency_links,
```

Comment:
This would be a regression of pip-tools support of `process_dependency_links` with older pip version.
I'd rather have this argument conditionally on `pip<19`.

Something like:
```
finder_kwargs = {
    "find_links": pip_options.find_links,
    "index_urls": index_urls,
    "trusted_hosts": pip_options.trusted_hosts,
    "allow_all_prereleases": pip_options.pre,
    "session": self.session,
}

if pkg_resources.parse_version(pip.__version__) < pkg_resources.parse_version('19.0'):
    finder_kwargs["process_dependency_links"] = pip_options.process_dependency_links

self.finder = PackageFinder(**finder_kwargs )
```
Or something cleaner using _compat if you think of something.

Reply:
Good point! Thanks.

Reply:
I've added a backward compatibility. Your code looks good to me, so i've used it.